Template/Template.csproj

```diff
@@ -32,6 +32,20 @@
         <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile"/>
     </ItemGroup>
 
+    <ItemGroup>
+        <!-- https://github.com/EvaisaDev/UnityNetcodePatcher -->
+        <!-- Allows to patch elements like networked behaviours, RPCs, etc. -->
+        <!-- Also, without that, the CI/CD will fail because the game's assemblies are not findable -->
+        <PackageReference Include="Evaisa.NetcodePatcher.MSBuild" Version="3.*" PrivateAssets="all"/>
+        <NetcodePatch Include="$(TargetPath)"/>
+    </ItemGroup>
```

Comment:
Remove lines 35-41 of this file - these are not necessary when using the CLI

Reply:
Removed